src/components/InformationCheckBox/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { useTheme } from '@mui/material';
+
+const getInfoCheckboxBackgroundColor = (variant: string) => {
+  const theme = useTheme();
+  switch (variant) {
+    case 'info':
+      return theme.palette.colors.lightL2;
+    case 'warning':
+      return theme.palette.colors.warning15;
+    case 'error':
+      return theme.palette.colors.error60;
+    default:
+      return theme.palette.colors.lightL2;
+  }
+};
+
+export default getInfoCheckboxBackgroundColor;
```

Comment:
does this need to be exported default? why have a utils file just to export default?